pixeltable/exprs/column_ref.py

```diff
@@ -170,6 +167,20 @@ def find_embedding_index(
             idx_info = embedding_idx_info
         return idx_info
 
+    def recompute(self, cascade: bool = True, errors_only: bool = False) -> catalog.UpdateStatus:
```

Comment:
I'd suggest the signature
```
def recompute(self, *, cascade: bool = True, errors_only: bool = False) -> catalog.UpdateStatus:
```
i.e., make `cascade` and `errors_only` kwargs-only parameters to match `Table.recompute_columns()`

Reply:
Done